__posts/2022-06-02-release-2.2.0.md

```diff
@@ -30,15 +30,15 @@ For more help with installation see the [installation instructions in the User G
 
 ## Notable new additions
 
- - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. (PR #3415)
+ - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. This new feature allows one to run per-frame analysis on arbitrary subsets of trajectory frames. (PR #3415)
```

Comment:
We should add the user github names like we've been doing in previous release posts.

Reply:
added

Reply:
You mean for each new thing? In which case I misunderstood you — I thought you only meant the "new contribs" list that was added based on your other comment.

At this stage having detailed features with names is a "nice to have".